analyses/doublet-detection/environment.yml

```
@@ -0,0 +1,14 @@
+name: openscpca-doublet-detection
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
```

Add platforms for easier `conda-lock` usage and slightly smaller files (we don't want the default list that includes windows)
```suggestion
platforms:
  - linux-64
  - osx-64
  - osx-arm64
dependencies:
```